source/common/router/router.h

```diff
@@ -179,22 +160,22 @@ class FilterUtility {
  */
 class FilterConfig {
 public:
-  FilterConfig(const std::string& stat_prefix, const LocalInfo::LocalInfo& local_info,
+  FilterConfig(Stats::StatName stat_prefix, const LocalInfo::LocalInfo& local_info,
```

Comment:
Is it better to pass StatName by value or reference? I'm not sure how big it is. Same elsewhere.

Reply:
It's better to pass by value as it's only a single pointer.